plugins/guisupport/guisupport.cpp

```diff
@@ -82,6 +84,14 @@ Q_DECLARE_METATYPE(QPixelFormat::ByteOrder)
 Q_DECLARE_METATYPE(QPixelFormat::ColorModel)
 Q_DECLARE_METATYPE(QPixelFormat::TypeInterpretation)
 Q_DECLARE_METATYPE(QPixelFormat::YUVLayout)
+Q_DECLARE_METATYPE(Qt::MouseButton)
+Q_DECLARE_METATYPE(QFlags<Qt::MouseEventFlag>)
+Q_DECLARE_METATYPE(QTouchEvent::TouchPoint)
+Q_DECLARE_METATYPE(QList<QTouchEvent::TouchPoint>)
+Q_DECLARE_METATYPE(Qt::TouchPointState)
+Q_DECLARE_METATYPE(QFlags<QTouchEvent::TouchPoint::InfoFlag>)
+Q_DECLARE_METATYPE(QFlags<QTouchDevice::CapabilityFlag>)
```

Comment:
This one is declared as Q_FLAG, which should auto-declare a metatype. So that shouldn't be needed.

Reply:
You're right, but somehow it doesn't compile without it (`qt_metatype_id’ is not a member of ‘QMetaTypeId<QFlags<QTouchDevice::CapabilityFlag> >`)